OracleIdentityGovernance/samples/externalDB/docker-compose.yaml

```diff
@@ -78,9 +86,13 @@ services:
       - ADMIN_HOST=${DC_HOSTNAME}
       - ADMIN_PORT=7001
       - ADMIN_PASSWORD=${DC_ADMIN_PWD}
-      - adminhostname=${DC_HOSTNAME}
-      - adminport=7001
       - MANAGED_SERVER=oim_server1
-      - server=oim_server1
+      - MS_HOST=${DC_HOSTNAME}
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://oimms:14000/identity"]
```

Comment:
If you're allowing the hostname to change, wouldn't this test need to change as well?